contracts/kernel/Kernel.sol

```diff
@@ -12,16 +12,26 @@ import "../factory/AppProxyFactory.sol";
 
 
 contract Kernel is IKernel, KernelStorage, Petrifiable, IsContract, VaultRecoverable, AppProxyFactory, ACLSyntaxSugar {
-    // Hardcode constant to save gas
-    //bytes32 public constant APP_MANAGER_ROLE = keccak256("APP_MANAGER_ROLE");
-    //bytes32 public constant DEFAULT_VAULT_APP_ID = apmNamehash("vault");
+    /* Hardcoded constants to save gas
+    bytes32 public constant APP_MANAGER_ROLE = keccak256("APP_MANAGER_ROLE");
+    bytes32 public constant DEFAULT_ACL_APP_ID = apmNamehash("acl");
+    bytes32 internal constant DEFAULT_VAULT_APP_ID = apmNamehash("vault");
+    */
     bytes32 public constant APP_MANAGER_ROLE = 0xb6d92708f3d4817afc106147d969e229ced5c46e65e0a5002a0d391287762bd0;
-    bytes32 public constant DEFAULT_VAULT_APP_ID = 0x7e852e0fcfce6551c13800f1e7476f982525c2b5277ba14b24339c68416336d1;
+    bytes32 public constant DEFAULT_ACL_APP_ID = 0xe3262375f45a6e2026b7e7b18c2b807434f2508fe1a2a3dfb493c7df8f4aad6a;
```

Comment:
Moved the `ACL_APP_ID` to here, as it's really the _default_ ACL ID (and also isn't needed anywhere else).